Move pam_oar_adopt back to the oar-node package.
pam_oar_adopt is indeed only required on OAR nodes.
This fixes bug #1099442.
Index: oar/Makefiles/common.mk
===================================================================
--- oar.orig/Makefiles/common.mk
+++ oar/Makefiles/common.mk
@@ -1,19 +1,14 @@
 MODULE=common
 SRCDIR=sources/core
 
-OARSH_DIR := $(if $(OAR_CGV1),oarsh-legacy,oarsh)
-
 OARDIR_BINFILES = $(SRCDIR)/tools/$(OARSH_DIR)/oarsh_shell.in \
 	          $(SRCDIR)/tools/$(OARSH_DIR)/oarsh.in \
                   $(SRCDIR)/qfunctions/oarnodesetting \
 		  $(SRCDIR)/tools/sentinelle.pl
 
-SBINDIR_FILES=$(SRCDIR)/tools/$(OARSH_DIR)/pam_oar_adopt
-
 MAN1DIR_FILES = $(SRCDIR)/man/man1/oarsh.1 \
 	       $(SRCDIR)/man/man1/oarprint.1
-MAN8DIR_FILES = $(SRCDIR)/man/man8/pam_oar_adopt.8 \
-	       $(SRCDIR)/man/man8/oarnodesetting.8
+MAN8DIR_FILES = $(SRCDIR)/man/man8/oarnodesetting.8
 
 SHAREDIR_FILES = $(SRCDIR)/tools/oar.conf.in \
                    $(SRCDIR)/tools/oarnodesetting_ssh.in \
Index: oar/Makefiles/node.mk
===================================================================
--- oar.orig/Makefiles/node.mk
+++ oar/Makefiles/node.mk
@@ -3,14 +3,16 @@ SRCDIR=sources/core
 
 SBINDIR_FILES=$(SRCDIR)/tools/oarnodecheck/oarnodecheckrun.in \
 		$(SRCDIR)/tools/oarnodecheck/oarnodechecklist.in \
-		$(SRCDIR)/tools/oarnodecheck/oarnodecheckquery.in
+		$(SRCDIR)/tools/oarnodecheck/oarnodecheckquery.in \
+		$(SRCDIR)/tools/$(OARSH_DIR)/pam_oar_adopt
 
 SHAREDIR_FILES= $(SRCDIR)/scripts/prologue \
 		$(SRCDIR)/scripts/epilogue \
 		$(SRCDIR)/tools/sshd_config.in \
 		$(SRCDIR)/scripts/oar-node-service
 
-MAN8DIR_FILES = $(SRCDIR)/man/man8/oarnodecheckrun.8
+MAN8DIR_FILES = $(SRCDIR)/man/man8/oarnodecheckrun.8 \
+				$(SRCDIR)/man/man8/pam_oar_adopt.8
 
 INITDIR_FILES = setup/init.d/oar-node.in
 
Index: oar/Makefiles/shared/shared.mk
===================================================================
--- oar.orig/Makefiles/shared/shared.mk
+++ oar/Makefiles/shared/shared.mk
@@ -7,6 +7,8 @@ export OARDO_UNINSTALL = $(MAKE) -f Make
 SHARED_INSTALL   = $(MAKE) -f Makefiles/shared/common_target.mk install
 SHARED_UNINSTALL = $(MAKE) -f Makefiles/shared/common_target.mk uninstall
 
+OARSH_DIR := $(if $(OAR_CGV1),oarsh-legacy,oarsh)
+
 # Detect if system is using systemd
 SYSTEMD_INIT? = $(shell if [ `ps --no-headers -o comm 1` = "systemd" ]; then \
 				echo "true"; \
